lib/src/model.dart

```diff
@@ -4287,10 +4328,7 @@ abstract class ModelElement extends Canonicalization
   ///
   String _stripHtmlAndAddToIndex(String rawDocs) {
     if (!config.injectHtml) return rawDocs;
-    final templateRegExp = new RegExp(
-        r'[ ]*{@inject-html\s*}([\s\S]+?){@end-inject-html}[ ]*\n?',
-        multiLine: true);
-    return rawDocs.replaceAllMapped(templateRegExp, (match) {
+    return rawDocs.replaceAllMapped(htmlRegExp, (match) {
```

Comment:
Heh.  I guess I haven't been paying too much attention to renaming things I cut and paste...